basyx.common/basyx.filerepository-backend/src/test/java/org/eclipse/digitaltwin/basyx/core/filerepository/backend/FileRepositoryTestSuite.java

```diff
@@ -149,5 +129,27 @@ public void doesNotExists() {
 		
 		assertFalse(fileNotExists);
 	}
+	
+	protected abstract FileRepository getFileRepository();
```

Comment:
The rule of public/priority hierarchy doesn't apply to the variable declaration.
So please move this to the top.